### PR TITLE
CY-2245 Fix a Docker container create/start cmd

### PR DESCRIPTION
**Pull request by mateumann**

### CODE REVIEW

**Comment by mateumann** [comment]
Staging docs (modified page) https://docs.cloudify.co/staging/CY-2245-docker-run-cmd/install_maintain/installation/manager-image/

**Comment by tehasdf** [comment]
lgtm
ping @ofercloudify

File: content/install_maintain/installation/manager-image.md [hunk 1]
```diff
@@ -44,16 +44,20 @@ Make sure that your environment meets the [prerequisites]({{< relref "install_ma
         This Docker image is designed to run on a remote instance, not on a docker installation on your local machine.
```
Comment (ofercloudify) [inline content/install_maintain/installation/manager-image.md]: The whole Docker section should be indented to the same level of the "Setup an Instance of a Cloudify Manager Image".
Originally when we asked the user to download the file and then load the container it maded sense to group them. now it doesnt make sense anymore.

File: content/install_maintain/installation/manager-image.md [hunk 2]
```diff
@@ -44,16 +44,20 @@ Make sure that your environment meets the [prerequisites]({{< relref "install_ma
...
-        
-        {{< highlight bash >}}
-        `sudo docker run --name cfy_manager_local -d --restart unless-stopped -v /sys/fs/cgroup:/sys/fs/cgroup:ro --tmpfs /run --tmpfs /run/lock --security-opt seccomp:unconfined --cap-add SYS_ADMIN -p 80:80 -p 8000:8000 cloudifyplatform/premium-cloudify-manager-aio:latest`
-        {{< /highlight >}}
+
+            {{< highlight bash >}}
+docker run --name cfy_manager_local -d --restart unless-stopped \
+  -v /sys/fs/cgroup:/sys/fs/cgroup:ro --tmpfs /run --tmpfs /run/lock \
+  --security-opt seccomp:unconfined --cap-add SYS_ADMIN \
+  -p 80:80 -p 5671:5671 -p 53333:53333 \
```
[the remark this answers is not in the document]
Comment (mateumann) [reply content/install_maintain/installation/manager-image.md]: No, not really. I do not know the details behind `:443` and `:5672` being exposed by this docker image but no actual process inside the container is listening on either one.

[the remark this answers is not in the document]
Comment (mateumann) [reply content/install_maintain/installation/manager-image.md]: As a result of this investigation @tehasdf created this one: https://github.com/cloudify-cosmo/cloudify-manager-install/pull/754

[the remark this answers is not in the document]
Comment (ofercloudify) [reply content/install_maintain/installation/manager-image.md]: @tehasdf 
5671 should be Rabbit listening for agents. Is this turned off on the Docker image?
I suppose that setting the UI to accept connections over SSL is configurable on the docker as well. if so, we need that port open.
What am I missing?

[the remark this answers is not in the document]
Comment (qooban) [reply content/install_maintain/installation/manager-image.md]: @ofercloudify @mateumann I'm not very familiar with Cfy docker image, but I suspect that no actual process is listening to `:443` when Cloudify works with SSL mode disabled. If Cloudify Manager would be working in SSL mode, then we should probably expose port 443. The question is if anybody changes that docker image to have Cloudify Manager working with SSL mode enabled or is it somehow configurable before running the container?

[the remark this answers is not in the document]
Comment (mateumann) [reply content/install_maintain/installation/manager-image.md]: Wrapping it up.

`:5672` is not an issue – Docker image should not expose it (and will not) neither should we talk about it in docs.

`:443` could be handled by manager's HTTP server but that would require some additional steps.  These could be one of the following:

- modification of `/etc/cloudify/config.yaml` and issuing `cfy_manager configure` 
- running `cfy ssl enable`

So, I decided to stick to this example and maybe provide some note about HTTP(S) port is configuration dependent.  I thought about doing it in another PR, but why wait :)

File: content/install_maintain/installation/manager-image.md (hunk 1 above)
Comment (mateumann) [reply content/install_maintain/installation/manager-image.md]: Are you sure?  _Docker_ section is on the same level as _OpenStack_.  Both are a second step of _Setup an Instance of a Cloudify Manager Image_ section.

Q: What would you like to do about _OpenStack_ section?  Leave it as is or change the indentation as well?

Q: What would you like to do about third step of _Setup an Instance of a Cloudify Manager Image_ section (`To use Cloudify Manager from the terminal using the Cloudify CLI, run the following command with your instance details.`)?

Comment (ofercloudify) [reply content/install_maintain/installation/manager-image.md]: Closing this one, and will handle the indentation as a separate PR.

**Pull request merged**